commands/poll.js

```diff
@@ -0,0 +1,50 @@
+const { prototype } = require("../modules/DiscordClientAwareScheduler");
```

Comment:
minor: why do u need this scheduler for poll

Reply:
cuz copy paste lolol